src/frontend/src/pages/Login.tsx

```diff
@@ -74,7 +83,9 @@ export default function Login() {
                color="primary"
                variant="contained"
                style={btnstyle}
-               onClick={() => sendData(user, password)}
+               onClick={() => {
+                  sendData();
+               }}
```

Comment:
This can be simplified to `onClick={sendData}` if you want.

Reply:
Changed